iohub/ngff_utils.py

```diff
@@ -0,0 +1,341 @@
+import contextlib
+import inspect
+import io
+import itertools
+import multiprocessing as mp
+from functools import partial
+from pathlib import Path
+from typing import Tuple
+
+import click
+import numpy as np
+from numpy.typing import DTypeLike
+
+from iohub.ngff import Position, open_ome_zarr
+from iohub.ngff_meta import TransformationMeta
+
+
+def create_empty_hcs_zarr(
```

Comment:
I am thinking about how this function is different from the standard `open_ome_zarr`. I think a better name for it may be `extend_hcs_zarr` since it adds positions and channels if they do not exist? This functionality is not evident from the current name `create_empty_hcs_zarr`.